Panel.c

```diff
@@ -421,7 +421,7 @@ bool Panel_onKey(Panel* this, int key) {
 
       case KEY_CTRL('E'):
       case '$':
-         this->scrollH = MAXIMUM(this->selectedLen - this->w, 0);
+         this->scrollH = MAXIMUM((int)this->selectedLen - this->w, 0);
```

Comment:
This could be improved to a `CLAMP`.

Reply:
This `int` cast suppresses the potential overflow of `this->selectedLen`. I didn't see code anywhere else guarding against `selectedLen` overflow (`> INT_MAX`) so I was hesitant to patch this one with a simple downcast.